server/api/controllers/liveSermon/scheduler-live-sermon.js

```diff
@@ -0,0 +1,60 @@
+const cron  = require('node-cron');
+
+// To change the isPublished value in the database from false to true
+const onLiveButon = async () => {
+  try{
+    var mongoClient = LiveSermon.getDatastore().manager.client;
+    var results = await mongoClient.db('hmcchk_db_test')
```

Comment:
Correct me if im wrong, but this seems to only work with the test db. it is better to use environment variables instead of calling the DB name directly. 

@ksuhartono97 @albertparedandan  please advice what the env variable name is in the prod server